src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -80,6 +86,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
         $filenames = $input->getArgument('filename');
         $showDeprecations = $input->getOption('show-deprecations');
+        $this->format = $input->getOption('format');
+
+        if ('github' === $this->format && !class_exists(GithubActionReporter::class)) {
+            throw new \InvalidArgumentException('The "github" format is only available since "symfony/console" >= 5.3.');
+        }
+
+        if (null === $this->format) {
```

Comment:
format cannot be `null` here because of https://github.com/symfony/symfony/blob/82e714812644ef5d4285d782b0143a4162c478d3/src/Symfony/Bridge/Twig/Command/LintCommand.php#L59

you must remove the default `txt` value (or set it accordingly tot he env detection result)